packages/block-directory/src/store/resolvers.js

```diff
@@ -35,5 +35,7 @@ export const getDownloadableBlocks =
 			);
 
 			dispatch( receiveDownloadableBlocks( blocks, filterValue ) );
-		} catch {}
+		} catch ( error ) {
```

Comment:
```suggestion
		} catch {
```

Suggestion: We're not using the `error`, which can be omitted along with the enclosing parentheses.